tools/docker/builder/openwrt/Dockerfile

```diff
@@ -90,7 +90,7 @@ RUN cp feeds.conf.default feeds.conf \
     && echo "CONFIG_TARGET_${TARGET_SYSTEM}_${SUBTARGET}_${TARGET_PROFILE}=y" >> .config \
     && echo "CONFIG_PACKAGE_prplmesh${PRPLMESH_VARIANT}=y" >> .config \
     && if [ "$TARGET_PROFILE" = "DEVICE_NETGEAR_RAX40" ] ; then \
-        cat feeds/intel/rax40.profile >> .config ;\
+        grep -v -E 'CONFIG_SDK|CONFIG_MAKE_TOOLCHAIN' feeds/intel/rax40.profile >> .config ;\
```

Comment:
Regarding this I think we should just remove these from the rax40.profile instead of hacking it away here. Better to sync with John why these are exported and if we can remove them before we workaround this in our Dockerfile.

Reply:
I asked John about it, but he might not reply immediately so I'll merge this, we can still change it later if it turns out we can remove it from the profile file.